pkg/apis/infrastructure/v1alpha2/awscluster_types.go

```diff
@@ -42,77 +38,6 @@ type AWSClusterSpec struct {
 	SSHKeyName string `json:"sshKeyName,omitempty"`
 }
 
-// NetworkSpec encapsulates all things related to AWS network.
```

Comment:
Why move these? Seems they are only used within this file

Reply:
these got moved to keep the awscluster_types.go and awsmachine_types.go files to contain only the kubebuilder generated types. The supporting types live in types.go